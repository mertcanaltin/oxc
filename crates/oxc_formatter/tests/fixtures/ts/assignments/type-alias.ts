export type _Repeat<A extends any, N extends number, L extends List = []> =
  __Repeat<N, A, L> extends infer X
  ? Cast<X, List>
  : never