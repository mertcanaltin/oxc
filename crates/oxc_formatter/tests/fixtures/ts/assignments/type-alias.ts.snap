---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
export type _Repeat<A extends any, N extends number, L extends List = []> =
  __Repeat<N, A, L> extends infer X
  ? Cast<X, List>
  : never
==================== Output ====================
export type _Repeat<A extends any, N extends number, L extends List = []> =
  __Repeat<N, A, L> extends infer X ? Cast<X, List> : never;

===================== End =====================
